src/components/pages/Chat/MessageInterceptor.tsx

```diff
@@ -74,6 +78,13 @@ const MessageInterceptor = memo(
                         }
                     }
                 )
+
+                send(MetadataType.UserOnline, {
```

Comment:
This `send` happens only on subbing? It may not be enough. Is there a mechanism in this PR that sends `MetadataType.UserOnline` periodically? Or is it a next-iteration thing?

Reply:
Probably next iteration, I'll make a ticket for it